docs/modules/bme680.md

```diff
@@ -143,15 +143,13 @@ bme680.setup()
 bme680.startreadout(150, function ()
     T, P, H, G, QNH = bme680.read(alt)
     if T then
-        local Tsgn = (T < 0 and -1 or 1); T = Tsgn*T
-        print(string.format("T=%s%d.%02d", Tsgn<0 and "-" or "", T/100, T%100))
-        print(string.format("QFE=%d.%03d", P/100, P%100))
-        print(string.format("QNH=%d.%03d", QNH/100, QNH%100))
-        print(string.format("humidity=%d.%03d%%", H/1000, H%1000))
+        print(string.format("T=%.2f", T/100))
```

Comment:
I would prefer to leave this the way that it was as it works for both integer and float builds.

Reply:
I though integer version is more like a history. But ok, I revert the change.